snscrape/modules/telegram.py

```diff
@@ -200,4 +208,4 @@ def _cli_setup_parser(cls, subparser):
 
 	@classmethod
 	def _cli_from_args(cls, args):
-		return cls._cli_construct(args, args.channel)
+		return cls._cli_construct(args, args.channel)
```

Comment:
Still an undesired whitespace change. There should be a LF at the end of a (text-ish) file.

Reply:
(Looks like GitHub doesn't display this correctly on the PR page itself, only in the full diff: <https://github.com/JustAnotherArchivist/snscrape/pull/687/files#diff-7f40c11448f92ed2f5d1764136d372d15faa3d4da0272813e88478c4d8870a09L203>)